Attachments/adding-attachments.md

```diff
@@ -0,0 +1,251 @@
+## How to upload and attach files from a third party applications
+
+The basic flow of uploading files in Lime Field looks like this:
+
+1. Create a `File` entity
+2. Upload file content to that entity's endpoint
+3. Connect the file entity as attachment to another entity in the API, e.g. `Case`, `Unit`, `Contract`
+
+__Note__, you probably will need to ensure that CORS requests are allowed from the domain where your application is hosted. Contact __Lime Field__ about that.
+
+So, lets see how the uploads can be done as in a code example.
+
+### Setup and Helpers
+
+First we will define some basic varables and helper functions to make the example code more readable. Hopefully.
```

Comment:
`varables` --> `variables`, remove `Hopefully.` at the end of the line.

Reply:
I like `Hopefully` here. Gives a personal touch.